crates/sncast/src/helpers/configuration.rs

```diff
@@ -99,4 +246,49 @@ mod tests {
             "Failed to find snfoundry.toml - not found in current nor any parent directories"
         );
     }
+
+    #[test]
+    fn parse_config_happy_case_with_profile() {
```

Comment:
nit: Since the name of the function has been changed, can we also rename these tests to `load_config...`?

Reply:
done https://github.com/foundry-rs/starknet-foundry/pull/1644/commits/321691a0c92cb4c5890734e9a931c1fd2c806607